cmd/glasskube/cmd/update.go

```diff
@@ -117,9 +119,54 @@ func completeInstalledPackageNames(
 	return
 }
 
+func completeUpgradablePackageVersions(
+	cmd *cobra.Command,
+	args []string,
+	toComplete string,
+) ([]string, cobra.ShellCompDirective) {
+
+	var dir cobra.ShellCompDirective
+	config, _, err := kubeconfig.New(config.Kubeconfig)
+	if err != nil {
+		dir = cobra.ShellCompDirectiveError
+		return nil, dir
+	}
+	client, err := client.New(config)
+	if err != nil {
+		dir = cobra.ShellCompDirectiveError
+		return nil, dir
+	}
+	if len(args) == 0 {
+		return nil, dir
+	}
+	packageName := args[0]
+	var packageIndex repo.PackageIndex
+	if err := repo.FetchPackageIndex("", packageName, &packageIndex); err != nil {
+		return nil, dir
+	}
+	if len(args) == 0 {
```

Comment:
I think this condition could be extended to `len(args) != 1`. The reason is that the version flag should not be used when more than one package is upgraded.